recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb

```diff
@@ -12,6 +12,14 @@ inherit setuptools3
 
 DEPENDS += "python3-setuptools-scm-native"
 
+do_install_append() {
```

Comment:
How does this end up RPI specific? The conditional seems to only be on hard float. which indeed is always the case for armv8. Should this me a rpi append?

Reply:
yeah its 32bit armhf specific since thats what the binary is for. the use of that binary is for SOCs which are in rpi boards 
I kept it with wider cut. Although this recipe has microcontroller specific implementations but that's a runtme thing